homeassistant/components/purpleair/sensor.py

```diff
@@ -171,9 +165,9 @@ class PurpleAirSensorEntityDescription(
         value_fn=lambda sensor: sensor.uptime,
     ),
     PurpleAirSensorEntityDescription(
+        # This sensor is an air quality index for VOCs. More info at https://github.com/home-assistant/core/pull/84896
         key="voc",
-        name="VOC",
-        device_class=SensorDeviceClass.AQI,
```

Comment:
Why was the device class removed? The comment above and removing the device class doesn't add up.

Reply:
Now I think of it, it does not make sense to remove it, because AQI isn't bound to one index. #96062

Reply:
Yes, that was the conclusion in the PR referenced in the code comment above. Hence, removing the device class as a result of the discussion in that PR doesn't make sense without further explanation.

For the future, please don't mix adding entity translations and removing other entity features. If we need to remove something, that should be a separate PR and discussion.

Also, please start filling out the PR template correctly. Adding entity translations isn't code quality. It's a new feature. And fill out the rest of the check boxes as appropriate. Eg, have you followed the PR checklist, there's no code commented etc.